torntools/scripts/content/global/ttGlobal.js

```diff
@@ -754,8 +756,8 @@ function nukeReviveScript() {
 
 // Code for adding chat filter for People Box
 function addPeopleBoxFilter() {
-	if (document.find(".chat-box-people_SPbEh").findAll(".tt-chat-filter").length === 0) {
-		let peopleBox = document.find(".chat-box-people_SPbEh");
+	if (doc.findAll("div[class*='chat-box-people_SPbEh'] .tt-chat-filter").length === 0) {
```

Comment:
Why would you use `*=` and still use the full class? Either do `.class_xxx` or `[class*='class_']`.